test/FullRange.t.sol

```diff
@@ -102,17 +101,14 @@ contract TestFullRange is Test, Deployers, GasSnapshot {
         keyWithLiq = createPoolKey(token0, token2);
         idWithLiq = keyWithLiq.toId();
 
-        modifyPositionRouter = new PoolModifyPositionTest(manager);
-        swapRouter = new PoolSwapTest(manager);
-
         token0.approve(address(fullRange), type(uint256).max);
         token1.approve(address(fullRange), type(uint256).max);
         token2.approve(address(fullRange), type(uint256).max);
-        token0.approve(address(swapRouter), type(uint256).max);
-        token1.approve(address(swapRouter), type(uint256).max);
-        token2.approve(address(swapRouter), type(uint256).max);
+        token0.approve(address(router), type(uint256).max);
+        token1.approve(address(router), type(uint256).max);
+        token2.approve(address(router), type(uint256).max);
 
-        manager.initialize(keyWithLiq, SQRT_RATIO_1_1, ZERO_BYTES);
+        initPool(keyWithLiq.currency0, keyWithLiq.currency1, fullRange, 3000, SQRT_RATIO_1_1, ZERO_BYTES);
```

Comment:
this `fullRange` hook address doesnt seem to be in the `keyWithLiq` key before?

Reply:
yeah it is... in createPoolKey